powerauth-java-crypto/src/test/java/io/getlime/security/powerauth/crypto/lib/util/GenerateVectorDataTest.java

```diff
@@ -475,6 +476,7 @@ public void testSignatureValidationV2() throws Exception {
             KeyPair deviceKeyPair = keyGenerator.generateKeyPair();
             PrivateKey devicePrivateKey = deviceKeyPair.getPrivate();
 
+            final PowerAuthSignatureFormat signatureFormat = PowerAuthSignatureFormat.getFormatForSignatureVersion("2.0");
```

Comment:
I would prefer to use `2.1` as latest `2.x` version.

Reply:
This doesn't matter. I wanted to test that function for various inputs, but then I wrote a separate unit test which covers all variants.